docs/examples/app.rb

```diff
@@ -0,0 +1,2 @@
+Shoes.app(title: "Shoes", width: 600, height: 600,resizable: false) do
```

Comment:
is it ok to do this? docs/examples, i did this becoz i though it would be easy to show usage of them
i can show examples directory but we "might" change them sometime 
but we rarely touch docs folder.

Reply:
Definitely reasonable to add examples in the docs folder. We should figure out where to link them from. But it's fine to add them before we figure that out.